docs/quickstarts/packet.md

```diff
@@ -27,17 +27,19 @@ The guide uses [Amazon Route 53](https://aws.amazon.com/route53/) as a DNS provi
 
 ## Requirements
 
-* Basic understanding of Kubernetes concepts.
-* Packet account, Project ID and auth token (sometimes also referred to as [User Level API
-  key](https://www.packet.com/developers/docs/API/getting-started/)).
-* AWS account and IAM credentials (optional for Route53 DNS configuration).
-* AWS Route53 DNS Zone (registered Domain Name or delegated subdomain).
-* Terraform v0.12.x and [terraform-provider-ct](https://github.com/poseidon/terraform-provider-ct) v0.5.0
-  installed locally.
-* Local BGP enabled. More information on how to enable Local BGP for the Packet Project is found in
-  the [Packet support document](https://support.packet.com/kb/articles/bgp).
-* An SSH key pair for management access.
-* `kubectl` installed locally to access the Kubernetes cluster.
+* A Packet account with
+  [local BGP](https://www.packet.com/developers/docs/network/advanced/local-and-global-bgp/)
+  enabled.
+* A Packet project ID.
+* A Packet API key with access to the relevant project.
+* An AWS account.
+* An AWS
+  [access key ID and secret](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html).
+* An AWS Route 53 zone (can be a subdomain).
+* An SSH key pair for accessing the cluster nodes.
+* Terraform `v0.12.x` installed.
+* [terraform-provider-ct](https://github.com/poseidon/terraform-provider-ct) `v0.5.0` installed.
+* `kubectl` installed.
```

Comment:
As far as I'm concerned we should **ship the provider**, but this is out of scope for this PR. As long as our code expects a certain version (which it does), we should mention it. Trying to deploy using the code in this PR with `v0.4.0` fails.

Reply:
The comment was on the `kubectl` line, I was talking about `kubectl` :)

Reply:
Right, sorry. I have no strong opinion here. If you don't like the current situation, please suggest an alternative :-)

Reply:
Mention to use the same version as deployed would make sense.

But this shows another problem: there is no easy way to know which kubernetes version we are installing in order for users to use the same kubectl version.

IMHO, something like this makes sense:
```suggestion
* `kubectl` installed and version should be compatible with Kubernetes version being installed.
```

What do you think? If you don't like it, I'm ok. I think this is clearer, but no strong opinion

Reply:
Canonical link to instructions on installing kubectl of whatever version user wants. https://kubernetes.io/docs/tasks/tools/install-kubectl/

Reply:
I will add a **note** mentioning the `kubectl` versioning constraints. Ideally we should either have a concrete version mentioned which we automatically increment as part of the release process, *or* link to some "compatibility matrix" document. In any case I'd rather keep things as simple as possible here. As long as things will generally work we should avoid bombarding the user with extra info and special case in a quickstart guide.

Reply:
I think a mention but without explicit versions (like the suggestion I did) might be a good trade-off.

If that doesn't seem good to you, I'm fine not saying anything about the version too. As I said in my original comment and you just mentioned, it will be a pain to maintain right now.

Reply:
I've handled this. Thanks.